karma-sauce.conf.js

```diff
@@ -27,12 +27,6 @@ module.exports = function(config) {
       recordScreenshots: false,
       recordVideo: false
     },
-
-    browserDisconnectTimeout: 10000,
```

Comment:
this was causing  our test to hang, I did multiple test and we don't need this for now.